packages/code-studio/src/styleguide/ContextMenus.tsx

```diff
@@ -129,7 +130,9 @@ class ContextMenus extends Component {
           }}
         >
           Right Click Me
-          <ContextActions actions={delayedActions} />
+          <ContextActions
+            actions={(delayedActions as unknown) as ResolvableContextAction[]}
```

Comment:
I know this is just the style guide... but there's a better way to fix this:
- In `ContextActions.tsx`, change `actions` in `ContextActionsProps` to allow for `ResolvableContextAction | ResolvableContextAction[]` (we check all through file if that prop is an array or not)
- Declare `delayedActions` as a `ResolvableContextAction`, get rid of casting as `Promise<ContextAction[]>` (e.g. `const delayedActions: ResolvableContextAction = ...`
- Remove casting here